docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+
+This SDK supports a minimum Android API of 23 or higher.
+Android apps can be written in Kotlin or Java 8 or higher.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
+
+### Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
+Follow the steps in [Enable the SDK](https://developer.paypal.com/sdk/in-app/android/#link-enablethesdk) to set up your account and application for PayPal payments.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used in place of a server SDK.
+
+### Technical steps - custom integration
+
+#### 1. Add the Payments SDK  to your app
+
+In your `build.gradle` file, add the following dependency:
+
+```groovy
+dependencies {
+   implementation "com.paypal.android:paypal-checkout:1.0.0"
+}
+```
+
+Add the following maven credentials to your `build.gradle`:
+```groovy
+maven {
+    url  "https://cardinalcommerceprod.jfrog.io/artifactory/android"
+    credentials {
+        username 'paypal_sgerritz'
+        password 'AKCp8jQ8tAahqpT5JjZ4FRP2mW7GMoFZ674kGqHmupTesKeAY2G8NcmPKLuTxTGkKjDLRzDUQ'
+    }
+}
+```
+
+#### 2. Create a PayPal button 
+
+Add a `PayPalButton` to your layout XML:
+
+```xml
+<com.paypal.android.checkout.paymentbutton.PayPalButton
+    android:id="@+id/payPalButton"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content" />
+```
+
+#### 3. Initiate the Payments SDK
+
+Create a `CoreConfig` using your client ID from the PayPal Developer Portal:
+
+```kotlin
+val config = CoreConfig("<CLIENT_ID>", environment = Environment.SANDBOX)
+```
+
+Configure a return URL using your application ID:
+
+```kotlin
+val returnUrl = "<APPLICATION_ID>" + "://paypalpay"
+```
+
+Create a `PayPalClient` to approve an order with a PayPal payment method:
+
+```kotlin
+val payPalClient = PayPalClient(application, config, returnUrl)
+```
+
+#### 4. Create an order
+
+When a user enters the payment flow, call `v2/checkout/orders` to create an order and obtain an order ID:
+
+```bash
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <ACCESS_TOKEN>' \
+--data-raw '{
+    "intent": "CAPTURE",
+    "purchase_units": [
+        {
+            "amount": {
+                "currency_code": "USD",
+                "value": "5.00"
+            }
+        }
+    ]
+}'
+```
+
+The `id` field of the response contains the order ID to pass to your client.
+
+#### 5. Approve the order through the Payments SDK
+
+When a user taps the PayPal button created above, approve the order using your `PayPalClient`.
```

Comment:
check out the comments I put on the card PR . our default integration should be such that the create order is called on the click of the PayPal button and not prior to it as that's not what we would recommend to most merchants).

Reply:
It was my understanding that we were not offering client-side order creation for the first iteration - has that changed?

Reply:
Got some further clarification on this - We aren't planning to offer client-side order creation, but the merchant can choose to create the order using their server integration at any point (before or after button click). Since we are currently only documenting the server integrations via curl commands, for easiest initial integration, it wouldn't really make sense to document that after the button click. However, merchants will have the ability to call their own order creation code on button click if they choose to.